### ISSUE TITLE
[bug] macOS dev window incorrectly launching within full-screen editors

### ISSUE THREAD

**Issue by stephenlrandall** [tauri-apps/tauri#4310]
### Describe the bug

On macOS, running `tauri dev` from a full-screen editor will cause the launched window to appear within that full-screen editor:

<img width="1536" alt="Screen Shot 2022-06-09 at 12 43 13 PM" src="https://user-images.githubusercontent.com/41397226/172900393-3485ecd0-1b7f-4159-a4ea-9c516523ee1c.png">

This breaks the macOS window model and doesn't allow the launched application to be moved. The dev application and the full-screen window are stuck together in Mission Control:

<img width="202" alt="Screen Shot 2022-06-09 at 12 43 32 PM" src="https://user-images.githubusercontent.com/41397226/172900665-a0660894-4b7e-4566-b4a3-67c54e4edde1.png">

### Reproduction

On macOS, open an editor with an integrated terminal like VS Code and run `tauri dev`. The launched application will be on top of the editor, unable to be moved.

### Expected behavior

I would expect a new application to go to an open desktop like any other launched macOS application does (and even like the `tauri build` disk image window).

### Platform and versions

```shell
Environment
  › OS: Mac OS 12.4.0 X64
  › Node.js: 18.2.0
  › npm: 8.9.0
  › pnpm: Not installed!
  › yarn: Not installed!
  › rustup: 1.24.3
  › rustc: 1.60.0
  › cargo: 1.60.0
  › Rust toolchain: stable-x86_64-apple-darwin 

Packages
  › @tauri-apps/cli [NPM]: 1.0.0-rc.13
  › @tauri-apps/api [NPM]: 1.0.0-rc.6
  › tauri [RUST]: 1.0.0-rc.8,
  › tauri-build [RUST]: 1.0.0-rc.7,
  › tao [RUST]: 0.8.4,
  › wry [RUST]: 0.15.1,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../build
  › devPath: http://localhost:3000/
  › framework: Svelte

App directory structure
  ├─ tauri
  ├─ node_modules
  ├─ static
  ├─ .svelte-kit
  ├─ build
  ├─ .git
  └─ src
```

**Comment by islandryu** [comment]
This may be due to the tao repository.
The window is to be activated after it is opened.
I will work on this.

**Comment by bicarlsen** [comment]
This also disables the ability to open the devtools, some hover actions (e.g. showing tooltips), and system interactions (e.g. opening a file chooser).

**Comment by thisislvca** [comment]
This has already been solved a while back, don't remember which version and couldn't find it. @FabianLars can you close this (sorry for the ping lol)

**Comment by FabianLars** [comment]
yep, seems to work for me as well. really can't remember a fix on our side, maybe it was a macos or vscode change 🤔
